File: 0237_Delete Node in a Linked List/DeleteNodeinaLinkedList.cs
/**
 * Definition for singly-linked list.
 * public class ListNode {
 *     public int val;
 *     public ListNode next;
 *     public ListNode(int x) { val = x; }
 * }
 */
public class Solution {
    public void DeleteNode (ListNode node) {
        if (node == null || node.next == null) return;
        var next = node.next;
        node.next = next.next;
        node.val = next.val;
    }
}

File: 0237_Delete Node in a Linked List/DeleteNodeinaLinkedList.py
# Definition for singly-linked list.
# class ListNode(object):
#     def __init__(self, x):
#         self.val = x
#         self.next = None

class Solution(object):
    def deleteNode(self, node):
        if node is None or node.next is None:
            return
        nextNode = node.next
        node.next = nextNode.next
        node.val = nextNode.val    